### ISSUE TITLE
Please add site https://moonlightnovel.com

### ISSUE THREAD

**Issue by bookimp** [dteviot/WebToEpub#1127]
### Provide URL for web page that contains Table of Contents (list of chapters) of a typical story on the site
https://moonlightnovel.com/novels/after-marrying-the-lord-of-the-underworld/

### Did you try using the Default Parser for the site?  If not, why not?
No, I didn't get the parser fields to try, it just had blank chapter boxes 

### If you don't have developer skills, can you ask a friend who does have them if they can do it for you?
I am not skilled in development nor do I know anyone who is.

**Comment by Kiradien** [comment]
Er, just to confirm, have you checked to confirm you're using the latest version of WebToEpub? Blank chapter boxes usually means it's using old logic, and when I checked locally it worked fine.

![image](https://github.com/dteviot/WebToEpub/assets/68683574/0149d3e4-eb82-4f7d-b2be-27cad964d760)

I do remember that site had a different layout in the past, I'm guessing the version you're using still has the old logic running. It might be worth uninstalling & reinstalling from the chrome or firefox store.

(copy & pasted from front page)
from Chrome Web Store
Start Chrome
Go to https://chrome.google.com/webstore/detail/webtoepub/akiljllkbielkidmammnifcnibaigelm
Click on the "Add to Chrome" button.
with Firefox
Start Firefox
Go to https://addons.mozilla.org/en-US/firefox/addon/webtoepub-for-baka-tsuki
Click on "Download anyway"

**Comment by bookimp** [comment]
Ok, apparently I was using v156 instead of v157 and now it works. I guess because I load a version from the google drive it doesn't auto-update.

**Comment by dteviot** [comment]
@bookimp 

Correct, the test builds don't auto update, the version from the store does.
Note, version in store is latest v156, and it does have support for moonlightnovel.
